packages/lesswrong/client/logging.ts

```diff
@@ -39,23 +40,25 @@ if (sentryUrl && sentryEnvironment && sentryRelease) {
 
 // Initializing sentry on the client browser
 
-userIdentifiedCallback.add(function identifyUserToSentry(user: UsersCurrent) {
-  // Set user in sentry scope
+userChangedCallback.add(function identifyUserToSentry(user: UsersCurrent | null) {
+  // Set user in sentry scope, or clear user if they have logged out
   Sentry.configureScope((scope) => {
-    scope.setUser({id: user._id, email: getUserEmail(user), username: user.username});
+    scope.setUser(user ? {id: user._id, email: getUserEmail(user), username: user.username} : null);
```

Comment:
There was a bug in these methods previously. `userIdentifiedCallback` would only be called if there was a currentUser so if a user logged out they wouldn't be unset in sentry or google analytics